 docker_image/snowshu.dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.7
```

Comment:
We should bump this up to 3.8 since we've added the walrus operator into the code which was first available in 3.8 I believe.

Reply:
true